VulkanHppGenerator.cpp

```diff
@@ -62,6 +62,53 @@ const std::string exceptionHeader(
   "\n"
   );
 
+std::string const exceptionClassesHeader = (
+  "#if defined(_MSC_VER) && (_MSC_VER == 1800)\n"
+  "# define noexcept _NOEXCEPT\n"
+  "#endif\n"
+  "\n"
+  "  class Error\n"
+  "  {\n"
+  "    public:\n"
+  "    virtual const char* what() const noexcept = 0;\n"
```

Comment:
I think, there's no need to have this function here, as both std::logic_error and std::system_error already provide that.
In order to make vk::Error a polymorphic type, it should be enough to have, for example, a virtual destructor here.